lua/mason-core/managers/std/init.lua

```diff
@@ -100,7 +100,11 @@ end
 function M.untar(file, opts)
     opts = opts or {}
     local ctx = installer.context()
-    ctx.spawn.tar {
+    local tar_cmd = ctx.spawn.tar
+    if platform.is.unix and not platform.is.linux then
+        tar_cmd = ctx.spawn.gtar
+    end
+    tar_cmd {
```

Comment:
This module and all others inside `mason-core.managers` are legacy and not used anywhere in mason.nvim. I think we can leave these as-is seeing as they're no longer supported anyways.